server/webapp/WEB-INF/rails/webpack/views/components/ellipsize/index.scss

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+$button_color: #2d6ca2;
```

Comment:
Can we define this under `theme.scss`?

Reply:
I was waiting for @naveenbhaskar  to provide css for it. Will submit new PR once I have it.